crates/task-impls/src/consensus2/mod.rs

```diff
@@ -1,10 +1,10 @@
-use async_broadcast::Sender;
-use hotshot_task::task::Task;
 use std::sync::Arc;
-use tracing::instrument;
 
+use async_broadcast::Sender;
```

Comment:
Seems like these changes might have been unnecessary?

Reply:
I ran `cargo fmt `which formatted the files this way